file/file-alfresco/src/test/java/uk/gov/justice/services/file/alfresco/requester/AlfrescoFileRequesterIT.java

```diff
@@ -87,6 +98,50 @@ public void shouldReturnResponseFromAlfresco() throws IOException {
     }
 
     @Test
+    public void shouldReturnPdfContentFromAlfresco() throws IOException {
```

Comment:
These tests are failing - it looks like you are not stubbing the Alfresco endpoint